internal/gengapic/gengapic_test.go

```diff
@@ -499,3 +499,55 @@ func Test_isLRO(t *testing.T) {
 		}
 	}
 }
+
+func Test_transportParse(t *testing.T) {
+	for _, tst := range []struct {
+		param        string
+		expectedOpts options
+		expectErr    bool
+	}{
+		{
```

Comment:
Maybe you also want to test more  error conditions (not just transport): missing `pkgPath`, `pkgName`, `outDir` and non-matching `modulePrefix`